frontend/src/metabase/components/DateRelativeWidget/DateRelativeWidget.module.css

```diff
@@ -0,0 +1,18 @@
+.shortcut:hover {
```

Comment:
Together with the markup change, where we replace `div` and `h6` with `fieldset` and `legend` to provide the automatic border cutoff

#### After
![Screenshot 2024-08-01 at 12 30 49 AM](https://github.com/user-attachments/assets/50e63116-c68d-407b-894e-cece20f8cdb3)

#### Before
![image](https://github.com/user-attachments/assets/a5b59064-b272-413f-9c81-5ff51b5bc24d)